tools/eslint/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@renovatebot/eslint-plugin",
+  "version": "0.0.5",
```

Comment:
Should we just reset it to 0.0.0 or 1.0.0 or something "placeholder" like that?

Reply:
It doesn't play any role to me 🤷